### ISSUE TITLE
Naming Convention in Maps

### ISSUE THREAD

**Issue by lheagy** [simpeg/simpeg#231]
We are inconsistent in naming conventions for maps
```
Maps.ExpMap
Maps.ActiveCells
Maps.Vertical1DMap
```

Do we want the trailing `Map` or not? (I vote not). Also, active cells to me sounds like we are grabbing a subset of the model (opposite). How about:
```
Maps.Exp
Maps.SurjActiveCells
Maps.SurjVertical1D
```
or replace `Surj` with `Surject`. To me, these are a bit more explicit in describing the action they are taking. Thoughts?

**Comment by rowanc1** [comment]
Also, that isn't a true surjection because the active cells are not in the
original set?

**Comment by lheagy** [comment]
Ah, true
'Maps.InjectActiveCells'

**Comment by lheagy** [comment]
closed with #265
